src/ChangeStream.php

```diff
@@ -48,16 +52,27 @@ public function __construct(Cursor $cursor, callable $resumeCallable)
         $this->key = 0;
     }
 
+    /**
+     * @see http://php.net/iterator.current
+     * @return mixed
+     */
     public function current()
     {
         return $this->csIt->current();
     }
 
-    public function getId()
+    /**
+     * @return MongoDB\Driver\CursorId
+     */
+    public function getCursorId()
     {
         return $this->csIt->getInnerIterator()->getId();
     }
 
+    /**
+     * @see http://php.net/iterator.mixed
```

Comment:
`iterator.key`. Looks like I made that typo in CachingIterator :smiley: 

Fixed in https://github.com/mongodb/mongo-php-library/commit/ba01a8cca36b084516ba82f30a3bcb378057f9e0.